.github/workflows/main.yml

```diff
@@ -0,0 +1,46 @@
+name: Triager Workflow
+
+on:
+  schedule:
```

Comment:
this need not be one single workflow, it can be two separate ones

Reply:
Sure will create two separate workflows for ci and bug scrub, thanks

Reply:
Done updated it in new commits